### DISCUSSION TITLE
unable to "npm init quasar"

### DISCUSSION THREAD

**Discussion by JongHyeonSong** [quasarframework/quasar#13937] [CLI - BEX mode]
1st step i installed "npm i -g @quasar/cli" with globally

and "npm init quasar" 

some text showup with "install create-quasar..." or someting

i suddenly stopped with "ctrl+c" becuase that folder was not to install

and then "npm init quasar" in other folder

error message

```
'create-quasar' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\siha1\Desktop\rr
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c C:\Users\siha1\AppData\Local\Temp\npx65766445389.cmd

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\siha1\AppData\Local\npm-cache\_logs\2022-07-12T22_20_52_939Z-debug-0.log
```

i tried "npm uninstall -g @quasar/cli" but npm command don't install create-quasar anymore...

"npm cache clean --force" dosent work

and i tried remove that file "npx65766445389.cmd" but thers no file in that path

i tried with yarn, and works well but i'd rather use npm 

any help?

add my info
```
$ quasar info

Operating System        Windows_NT(10.0.22000) - win32/x64
NodeJs                  16.16.0

Global packages
  NPM                   8.13.2
  yarn                  1.22.19
  @quasar/cli           1.3.2
  @quasar/icongenie     Not installed
  cordova               Not installed

Networking
  Host                  sjh
  Wi-Fi                 172.30.1.24
```

and  when i installed it by myself "npm install -g create-quasar" 
and then "npm init quasar" command use "create-quasar" well and scafolled everything well 
any problem in this way?

**Comment by metalsadman** [comment]
post your `quasar info` logs.

**Comment by b0ot** [comment]
I am having a similar issue currently.

On Mac
npm-init-quasar 
is asking for me to install the following packages: 'create-quasar'

I have been avoiding doing so, as it seems like that should have already been installed with 
npm i -g @quasar/cli
